.github/workflows/ci.yml

```diff
@@ -479,10 +479,17 @@ jobs:
         run: |
           CONTAINERD_RUNTIME=$TEST_RUNTIME make cri-integration
 
+      - name: Fix up for actuated
+        # https://github.com/containerd/containerd/pull/9920#issuecomment-2024823587
+        if: ${{ (matrix.os == 'actuated-arm64-4cpu-16gb') && (matrix.runc == 'crun') }}
+        run: |
+          echo "EXTRA_CRITEST_OPTIONS='--ginkgo.skip='runtime should support NamespaceMode_POD'" >> $GITHUB_ENV
```

Comment:
Sorry for that. Please use this one .`echo "EXTRA_CRITEST_OPTIONS=--ginkgo.skip=runtime should support NamespaceMode_POD" >> $GITHUB_ENV`

Reply:
REF: https://github.com/containerd/containerd/actions/runs/8949090549/job/24583037078?pr=10170

Reply:
Thanks, applied the fix